### PR TITLE
Implementation of new geOrchestra header

### PR DESCRIPTION
**Pull request by f-necas**
Update default header to point to the new one

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Oops ?

File: datahub/scripts/add-header-webcomponent.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 DATAHUB_DIR=${1:-/usr/share/nginx/html/datahub}
-SNIPPET="<script src='https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js'></script><geor-header></geor-header>"
+SNIPPET="<script src='https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js'></script><geor-header active-app='datahub' style='height:90px'></geor-header>"
```
Comment (tkohr) [inline datahub/scripts/add-header-webcomponent.sh:4]: Would be nice if there was a way we could use the values from the `default.properties` here, especially for the `headerScript`.

File: default.properties [hunk 2]
```diff
@@ -25,7 +25,9 @@ headerHeight=90
 # Header URL (can be absolute or relative)
 # If different from default value "/header/", adapt
 # security-proxy/targets-mapping.properties accordingly
```
Comment (tkohr) [inline default.properties:27]: We should probably update the comment here.

File: datahub/scripts/add-header-webcomponent.sh (hunk 1 above)
Comment (edevosc2c) [reply datahub/scripts/add-header-webcomponent.sh:4]: Then maybe this should be supported natively by datahub. Datahub having the functionality of reading a properties file in order to add a "header".

Comment (jahow) [reply datahub/scripts/add-header-webcomponent.sh:4]: For now we won't go that way, the "header" feature is still very much a geOrchestra thing. Let's see if other users need this as well, then we can revisit.

Comment (tkohr) [reply datahub/scripts/add-header-webcomponent.sh:4]: Yep, I guess for now we don't have an alternative than to repeat some values from the `default.properties`.

**Review by tkohr — APPROVED** [review]
Thanks, @f-necas ! Works fine, just updating the comment would be nice :-)

**Comment by fvanderbiest** [comment]
Thanks for all the work.
2 things here:

- you should remove the branch when the PR is merged (by clicking the button)
- you should take care of syncing your work with the `master` branch, which is used in the "non-docker" context. A cherry-pick is OK too. Please proceed.

**Comment by fvanderbiest** [comment]
Maybe you should add some lines about the header switch in the migration notes for the upcoming 23.1 or 24.0 release (whose name depends on the release date), in https://github.com/georchestra/georchestra/blob/master/migrations/master/README.md

**Pull request merged**